libbeat/cmd/instance/umask_windows.go

```diff
@@ -0,0 +1,25 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package instance
+
+import "github.com/elastic/go-sysinfo/types"
+
+func setUmask(newmask int) error {
+	// No way to set umask on Windows
+	return types.ErrNotImplemented
```

Comment:
This is an internal error. We should not reuse sentinal errors from 3rd party packages (packages vendored by beats).

Reply:
Thanks, wasn't sure about whether this was a good idea or not. Will change.